src/image.cpp

```diff
@@ -332,25 +332,38 @@ namespace Exiv2 {
         return type >= 1 && type <= 13 ;
     }
 
+
+    typedef std::pair<uint32_t,uint32_t> dict_t ;
+    typedef std::vector<dict_t>          dicts_t;
+    static  dicts_t                      dicts  ;
```

Comment:
Does this variable need to be static? it is only used in `Image::printIFDStructure` and in line 365 we are doing `push_back`, so that the vector can grow without limits.